src/lite-yt-embed.js

```diff
@@ -40,6 +39,19 @@ class LiteYTEmbed extends HTMLElement {
         // TODO: support dynamically setting the attribute via attributeChangedCallback
     }
 
+    fetchYoutubeAPI(cb) {
```

Comment:
Why not make this a promise?

Reply:
Yes, it could be.
But honestly, I don't see any blocker in this case.